echo/wavefront-tracing-logger.go

```diff
@@ -0,0 +1,168 @@
+package echo
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+
+	"github.com/labstack/echo"
+	"github.com/opentracing/opentracing-go"
+)
+
+//WfLogger custom implementation of Go standard Logger
+//Exposes the same functions as Standard Logger
+type WfLogger struct {
+	logger  *log.Logger
+	context echo.Context
+}
+
+// Writer custom writer 
+type Writer struct {
+	io.Writer
+	timeFormat string
+}
+
+func (w Writer) Write(b []byte) (n int, err error) {
+	return w.Writer.Write(append([]byte(time.Now().Format(w.timeFormat)), b...))
+}
+
+//NewWfLogger returns a new custom logger instance with TraceId and SpanId injected.
+//It exposes the same functions as the standard Golang Logger, with added function of injecting trace info in logs.
```

Comment:
Where is the trace info injected into logs?

Reply:
log prefix is extracted from context which is added by middleware in the processing. This is added as prefix too logger.  log.New(&Writer{os.Stdout, "2006-01-02 15:04:05 "}, fmt.Sprintf("%v", logPrefix), 0)